src/index.html

```diff
@@ -15,8 +15,95 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="img_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_items nav_link_1">
+            <a
+              href="index.html"
+              class="nav_links is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links nav_link_4"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav_items">
+            <a
+              href="index.html"
+              class="nav_links"
+            >
+              Video
+            </a>
```

Comment:
All navigation links currently point to `index.html`. If these links are supposed to navigate to different pages, you should update the `href` attributes accordingly.